package/yast2-schema.spec

```diff
@@ -35,6 +35,12 @@ BuildRequires:	yast2-online-update-configuration
 
 # yast2-core omited from some reason in buildservice, adding explicitly
 BuildRequires:  yast2-core
+
+# openSUSE does not contain the registration module
+%if %suse_version == 1315
```

Comment:
`%if !0%{?is_opensuse}`  is better as it will work also with the future SLE releases.